plugins/cost-insights/src/utils/formatters.ts

```diff
@@ -17,33 +17,36 @@
 import { DateTime, Duration as LuxonDuration } from 'luxon';
 import pluralize from 'pluralize';
 import { ChangeStatistic, Duration } from '../types';
-import { inclusiveEndDateOf, inclusiveStartDateOf } from '../utils/duration';
+import { inclusiveEndDateOf, inclusiveStartDateOf } from './duration';
 import { notEmpty } from './assert';
 
 export type Period = {
   periodStart: string;
   periodEnd: string;
 };
 
-export const costFormatter = new Intl.NumberFormat('en-US', {
-  style: 'currency',
-  currency: 'USD',
-});
+export const currencyFormatter = (currency: Intl.NumberFormat) => {
```

Comment:
Could you not combine this into the `createCurrency` that takes optional parameters for the additional options? It seems these are just wrapping that anyways.

Reply:
The purpose of these 2 functions are a bit different.
`createCurrencyFormat` is to create this complicated object structure, and for `currencyFormatter` to reset `minimumFractionDigits` and `maximumFractionDigits` to 0. 

I suggest to rename `currencyFormatter` together with `lengthyCurrencyFormatter` to make it more explicit.